client/app/controllers/projects.js

```diff
@@ -0,0 +1,28 @@
+import Controller from '@ember/controller';
+
+export default class ProjectsController extends Controller {
+  // projects for applicant to do
+  get applicantProjects () {
+    // handle filtering here, reference in template as this.applicantProjects
+    // todo if statuscode is "Save" or "Package Preparation"
+    return this.model.filter((project) => project.packages.some((projectpackage) => {
+      if (projectpackage.statuscode === 'Saved' || projectpackage.statuscode === 'Package Preparation') {
+        return true;
+      } return false;
+    }));
+  }
+
+  // projects for nyc planning to do
+  get planningProjects () {
+    return this.model.filter((project) => project.packages.some((projectpackage) => {
+      if (projectpackage.statuscode === 'Submitted' || projectpackage.statuscode === 'Under Review' || projectpackage.statuscode === 'Revision Required') {
+        return true;
+      } return false;
```

Comment:
One way to simplify this a little more would be to create a computed property on the project model. This computed property could calculate the "type" of project it is.

Reply:
In new branch, `17-...-v2`, left a TODO comment to fix this in the future but get this working for MVP.